client.py

```diff
@@ -5,57 +5,60 @@
 from discord.ext import commands
 
 import data
-from cogs import *
+
 
 class TTSBot(commands.Bot):
     def __init__(self):
-        super().__init__(
-            command_prefix='$',
-            intents=discord.Intents.default())
+        super().__init__(command_prefix="$", intents=discord.Intents.default())
         self.data = data.Data()
         self.set_env_vars()
 
     def set_env_vars(self):
         self.TOKEN, self.GUILD_NAME = self.data.get_env_vars()
-        if self.TOKEN == None or self.GUILD_NAME == None:
-            print("ERROR in .env file, double check format in top-level README.md")
+        if not all(self.TOKEN or self.GUILD_NAME):
+            print(
+                "ERROR in .env file, double check format in top-level"
+                " README.md"
+            )
             exit()
 
     async def setup_hook(self):
-        #Load cogs
-        for filename in os.listdir('./cogs'):
-            if filename.endswith('.py') and filename != '__init__.py':
+        # Load cogs
+        for filename in os.listdir("./cogs"):
+            if filename.endswith(".py") and filename != "__init__.py":
```

Comment:
Consider using [`pathlib.Path.glob()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob) here perhaps

Reply:
You got it boss man.